docs/learn/software_overview.md

```diff
@@ -1,53 +1,54 @@
-# Software Overview
+# Workflow & Software Overview
 
-Starting out in the CNC world can be very intimidating, even if you already know about 3D printing
-(which is a CNC machine, technically). Here, we will break down the different software components
-and tools, to give you a better understanding of some of the terminology, which gives you the
-language to find help. Also, we want you to have an overview of the area, so you can understand all
-of the parts, and how they work together.
+Starting out in the [CNC](https://en.wikipedia.org/wiki/Numerical_control) world can be very intimidating, even if you already know about 3D printing (which is a CNC machine, technically).  However, when broken down into manageable pieces it's easy to digest.  In the world of CNC, workflow and software are intrinsically linked. 
```

Comment:
I am agnostic on this.  To be fair I added this link for two reasons 1. Practice for markup 2.  Once someone asked me when I told them I was building a CNC machine they asked me what CNC stood for.  I embarrassingly forgot at the time.  Frankly if you are a person actually reading this documentation by the time you get here, you probably know what CNC stands for.

Reply:
Yeah. I think of CNC sort of like SCUBA, LASER, or RADAR. It is an acronym, but no one really cares what it is. This may just be me though. I get annoyed whenever there is an article about CNC and the first thing they do is tell us what it stands for. This is totally my pet peeve though. I have no expectation of others to also care about it.